Example/Source/Theme.swift

```diff
@@ -25,8 +25,8 @@ class Theme {
         // Navigation
         UINavigationBar.appearance().barTintColor = .brickBackground
         UINavigationBar.appearance().titleTextAttributes = [
-            NSForegroundColorAttributeName: Theme.textColorForNavigationTitle,
-            NSFontAttributeName: Theme.fontForNavigationTitle
+            NSAttributedStringKey.foregroundColor: Theme.textColorForNavigationTitle,
+            NSAttributedStringKey.font: Theme.fontForNavigationTitle
```

Comment:
you should be able to remove the `NSAttributedStringKey` and just use `.foregroundColor` and `.font`